book/book.tex

```diff
@@ -29,6 +29,11 @@
 \usepackage{kantlipsum}
 \usepackage{cleveref}
 
+\usepackage[version=4]{mhchem}
+
+\newcommand{\eqnref}{Eq.~\eqref}
+\newcommand{\figref}{Fig.~\ref}
```

Comment:
We should not define new commands to reference figures and equations. Instead you should just use `\eqref` and `\ref`.

Reply:
I'll work around the defaults without the special commands.